src/raw/RawModel.cpp

```diff
@@ -195,10 +201,10 @@ int RawModel::AddMaterial(
     const int textures[RAW_TEXTURE_USAGE_MAX],
     std::shared_ptr<RawMatProps> materialInfo,
     const std::vector<std::string>& userProperties) {
-      auto it = materialIdToIndex.find(id);
-      if (it != materialIdToIndex.end()) {
-        return it->second;
-      }
+  auto it = materialIdToIndex.find(id);
```

Comment:
Not a big deal, but this whitespace change went to far the other way?

Reply:
This is actually correct - if you look at it with more context, the lines right above are the function declaration.